externs/browser/w3c_indexeddb.js

```diff
@@ -192,7 +206,7 @@ IDBDatabase.prototype.objectStoreNames;
 
 /**
  * @param {string} name The name of the object store.
- * @param {Object=} opt_parameters Parameters to be passed
+ * @param {!Object|!IDBObjectStoreParameters=} opt_parameters Parameters to be passed
```

Comment:
My bad since I suggested this.
Disallowing nulls here causes unnecessary and unhelpful breakage.
I've changed the '!'s to '?'s in my imported copy and am continuing to test that.